.github/actions/find-runner.js

```diff
@@ -0,0 +1,10 @@
+const { Octokit } = require("@octokit/action");
+
+const octokit = new Octokit();
+
+get_async()
+
+async function get_async () {
+  const result = await octokit.request('GET /repos/Homebrew/homebrew-core/actions/runners')
+  console.log("Result: %s", result.data);
```

Comment:
Is any of this data potentially sensitive?

Reply:
Nope, looks harmless:
```
Result: { total_count: 1, runners: [Array] }
7
Result: {"total_count":1,"runners":[{"id":99,"name":"linux-self-hosted-1","os":"linux","status":"online","busy":false,"labels":[{"id":7,"name":"self-hosted","type":"read-only"},{"id":8,"name":"Linux","type":"read-only"},{"id":9,"name":"X64","type":"read-only"},{"id":10,"name":"linux-self-hosted-1","type":"custom"}]}]}
```